_includes/global_fields.md

```diff
@@ -4,6 +4,7 @@ A set of common fields that appear on all resources is included first, for refer
 
 | Name          | Type      | Description
 |-----------    |-----------|-----------|--------------
+|id 			| string    | A string representing a local identifier for the resource on the local system.  It may be a string, integer, guid, etc.  This value MUST be suitable for use in a helper function such as add_tags, add_lists to identify the corresponding resource
```

Comment:
I don't mind that certainly. It's more HAL-like for sure. But we've tried to avoid using links in the content for POSTs so far too.